stor_swift/tests/test_cli_swift.py

```diff
@@ -22,7 +22,7 @@ def setUp(self):
         patcher.start()
 
     @contextlib.contextmanager
-    def assertOutputMatches(self, exit_status=None, stdout='', stderr=''):
+    def assertOutputMatches(self, exit_status=None, stdout='', stderr=''):  # pragma: no cover
```

Comment:
add a test for this brah

Reply:
It is just a helper function used by tests in test_cli_swift. Since some stuff was moved here from test_cli.py, I also had to move this function here. This currently resembles the same func from test_cli.py but I can scope it to the facets that are used by swift tests:
```
    def assertOutputMatches(self, exit_status=None, stdout='', stderr=''):
        patch = mock.patch('sys.stderr', new=six.StringIO())
        self.addCleanup(patch.stop)
        patch.start()
        try:
            yield
        except SystemExit as e:
            self.assertEquals(e.code, int(exit_status))
        self.assertEquals(sys.stdout.getvalue(), '', 'stdout')
        self.assertRegexpMatches(sys.stderr.getvalue(), stderr, 'stderr')
```
Don't have to use no pragma then. Is this ok?

Reply:
the reason you need pragma no cover is because you're missing a test of a test.  Just need something like:

```
with self.assertRaises(AssertionError):
    with self.assertOutputMatches(exit_code=10):
        self.run('stor blahrg')
with self.assertRaises(AssertionError):
    with self.assertOutputMatches(stdout='mystdout'):
        self.run('stor -h')
```

it's just confirming that your matcher can actually fail :)

Reply:
Agreed, changed